### DISCUSSION TITLE
It is difficult to find the "New Conversation" button when the number of conversations increases

### DISCUSSION THREAD

**Discussion by CNSeniorious000** [anse-app/anse#19] [Ideas]
I think we can place the "New Conversation" button at the top of the list, or even more radically, to the right of the header "CONVERSATIONS", like here:

![image](https://github.com/anse-app/anse/assets/74518716/a7ab739d-c910-4dcf-b39c-6afeff25811c)

Maybe this is efficient especially before keyboard shortcuts being implemented.

**Comment by ddiu8081** [comment]
Yes, I plan to update the sidebar design soon to allow it to carry more conversations, as well as to be able to support the ability to group conversations in the plan.

**Comment by ddiu8081** [comment]
Moved it to the top. Thank you!

**Reply by CNSeniorious000** [reply]
The new release is awesome! Good job🤩
